janusgraph-core/pom.xml

```diff
@@ -64,6 +64,11 @@
             <artifactId>spatial4j</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
```

Comment:
@mbrukman I created #128 to track automating dependency license check as part of CI